src/assets/wise5/classroomMonitor/classroomMonitorComponents/shared/grading-helpers/grading-helpers.ts

```diff
@@ -0,0 +1,10 @@
+export function isComponentVisibleToStudent(nodeStatus: any, component: any): boolean {
+  let result = true;
+  if (nodeStatus.componentStatuses != null) {
+    const componentStatus = nodeStatus.componentStatuses[component.id];
+    if (componentStatus != null) {
+      result = componentStatus.isVisible;
+    }
+  }
+  return result;
+}
```

Comment:
This function has a lot of checks, which makes it difficult to read. It looks like we don't need the nodeStatus object here. What if we pass in the componentStatuses object with a default empty object? Can you also write a test for this function?
```suggestion
export function isComponentVisibleToStudent(
  componentStatuses: any = {},
  componentId: string
): boolean {
  const componentStatus = componentStatuses[componentId];
  if (componentStatus != null) {
    return componentStatus.isVisible;
  }
  return true;
}
```